features/src/test/scala/com/salesforce/op/utils/spark/RichDatasetTest.scala

```diff
@@ -234,6 +234,16 @@ class RichDatasetTest extends FlatSpec with TestSparkContext {
     in(ds) noneOf "f2" should ((x: Double) => x > 3.5)
   }
 
+  it should "only compare types and ignore nullability" in {
```

Comment:
Minor correction: lets replace `it` in this test case with `Spec(com.salesforce.op.utils.spark.RichDataType.getClass)`.

Reply:
Changed usage of "it" to reflect RichDataType.
Thanks